src/components/inputs/rich-text-input/editor.js

```diff
@@ -29,14 +32,23 @@ const Editor = props => {
     if (!doesExceedCollapsedHeightLimit && renderToggleButton) {
       setRenderToggleButton(false);
     }
-  }, [props.editor.value.document, renderToggleButton]);
+  }, [setRenderToggleButton, renderToggleButton]);
+
+  React.useEffect(() => {
+    updateRenderToggleButton();
+  }, [props.editor.value.document, updateRenderToggleButton]);
 
   return (
     <CollapsibleMotion
       minHeight={COLLAPSED_HEIGHT}
       isDefaultClosed={!props.defaultExpandMultilineText}
     >
       {({ isOpen, toggle, containerStyles, registerContentNode }) => {
+        // opens the input if it regains focus and it's closed
+        if (prevIsFocused !== props.isFocused && props.isFocused && !isOpen) {
```

Comment:
so if the focus has changed from not being focused to focused, and it's not open, we should toggle it open